api/design/resource.go

```diff
@@ -30,13 +30,14 @@ var _ = Service("resource", func() {
 			Attribute("name", String, "Name of resource", func() {
 				Default("")
 			})
-			Attribute("kind", String, "Kind of resource", func() {
-				Enum("task", "pipeline", "")
-				Default("")
-			})
+			Attribute("kinds", ArrayOf(String), "Kinds of resource")
+			Attribute("tags", ArrayOf(String), "Tags associated with a resource")
 			Attribute("limit", UInt, "Maximum number of resources to be returned", func() {
 				Default(100)
 			})
+			Attribute("exact", Boolean, "Find the exact resource", func() {
```

Comment:
you can use an `Enum` here I guess to restrict the input...`true` and `false`

Reply:
Since it is boolean do we still need to use enum ?

Reply:
if we pass anything other that, does it return an error?

Reply:
then if is fine

Reply:
Yes that would return an error

Reply:
Saying the type of exact should be boolean